### PR TITLE
improved style cleanup tool; some cosmetics

### PR DESCRIPTION
**Pull request by following5**

### CODE REVIEW

File: local/devel/style-cleanup.php [hunk 1]
```diff
@@ -9,22 +9,30 @@
...
+$exclude = array(
     'htdocs/cache',
     'htdocs/cache2',
     'htdocs/lib2/smarty',
     'htdocs/okapi',
     'htdocs/templates2/mail',
     'htdocs/var',
-    'htdocs/vendor'
-];
+    'htdocs/vendor',
```
Comment (teiling88) [inline local/devel/style-cleanup.php]: the last , is not necessary

**Comment by teiling88** [comment]
pls fix the last , after that it is ready to merge

File: local/devel/style-cleanup.php (hunk 1 above)
Comment (following5) [inline local/devel/style-cleanup.php]: It simplifies maintaining the list, especially when sorting entries. Nice habit from OKAPI coding style. ;)

**Comment by following5** [comment]
I'ts not broken. Does phpStorm complain about it?

**Pull request merged**